xdsl/backend/llvm/__init__.py

```diff
@@ -0,0 +1,182 @@
+import ctypes
+import subprocess
+
+import llvmlite
```

Comment:
Let's ignore this error for now
```suggestion
import llvmlite # pyright: ignore[reportMissingTypeStubs]
```

Reply:
Similarly below